src/Event/BundleManagerEvents.php

```diff
@@ -16,41 +16,9 @@
 
 namespace Pimcore\Bundle\AdminBundle\Event;
 
-final class BundleManagerEvents
+/**
+ * @TODO this class is only here for BC reasons and should be removed in Pimcore 12
```

Comment:
Shouldn't it be deprecated then?
```suggestion
 * @deprecated this class is only here for BC reasons and should be removed in Pimcore 12
```

Reply:
we want to avoid this intentionally as this was already deprecated on 10.6. https://github.com/pimcore/pimcore/pull/15468/files#diff-29364e2d0b002e03d5bb2e7e6b0b6c255943ddf7926d965527d01b8c48509c5eL19

Reply:
Its bad yes. But I think the developer should know that now he should use the one in the core, right? So the deprecated annotation would help.

Reply:
It will stay there for v11 version so shouldn't be a problem to keep using it.

Reply:
good point but it cannot be deprecated in a bug fix release, it likely would need the deprecated tag in the next latest minor (like what happened for `10.6`) before major, right?

Reply:
Why not adding it in 11.1? So the developer have more time to change it.

Reply:
Why can't it be deprecated in a bug fix release?

Reply:
because bugfixes are not suppose the change the running behavior of the application. we want to merge it in bugfix to avoid confusion with https://github.com/pimcore/pimcore/pull/15468/files#diff-29364e2d0b002e03d5bb2e7e6b0b6c255943ddf7926d965527d01b8c48509c5eL19

Reply:
But what does a deprecation change (except saying: please don't use me anymore)?

Reply:
the notice changes the context for running code...right?

Reply:
Hmm, that might be true for `trigger_deprecation()`, yes :thinking: 
But adding a `@deprecated` in the PHPDoc shouldn't change anything...
Anyway, I'm fine with deprecating it in `11.1`.